### PR TITLE
feat: collect realtime init container logs

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Add a init container to copy cyclone toolbox to other init containers, so that those init containers can report realtime logs to cyclone server.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: release/workflow-controller.yaml [hunk 1]
```diff
@@ -47,6 +47,7 @@ _config:
               "svn-resolver": "[[ registry_release ]]/cyclone-resolver-svn:[[ imageTagFromGitTag ]]",
               "image-resolver": "[[ registry_release ]]/cyclone-resolver-image:[[ imageTagFromGitTag ]]",
               "coordinator": "[[ registry_release ]]/cyclone-workflow-coordinator:[[ imageTagFromGitTag ]]",
+              "toolbox": "[[ registry_release ]]/cyclone-toolbox:[[ imageTagFromGitTag ]]",
```
Comment (supereagle) [inline release/workflow-controller.yaml]: Where is the code of `cyclone-toolbox`?

Comment (cd1989) [reply release/workflow-controller.yaml]: Another PR

**Pull request merged**